site.maintenance.aws-remove-all.yml

```diff
@@ -187,20 +199,29 @@
           with_items:
             - "{{ aws_s3_media_bucket }}"
             - "{{ aws_s3_secret_bucket }}"
-      when: remove_s3
+      when: remove_s3 and not dry_run_mode
+
+    - name: Query Launch templates
+      shell: |
+        aws ec2 describe-launch-templates \
+          --region '{{ aws_region }}' \
+          --filter "Name=tag:Project,Values={{ mageops_project }}" --filter "Name=tag:Environment,Values={{ mageops_environment }}"
```

Comment:
Filter using all tags, in the same way like it's done in `roles/cs.aws-ec2-cleanup/tasks/main.yml`